index.js

```diff
@@ -95,19 +164,11 @@ getMove = function (control) {
 };
 
 module.exports = defineProperties(getFn(), {
-	width: d.gs(process.stdout.getWindowSize ? function () {
-		return process.stdout.getWindowSize()[0];
-	} : function () {
-		return tty.getWindowSize ? tty.getWindowSize()[1] : 0;
-	}),
-	height: d.gs(getHeight = process.stdout.getWindowSize ? function () {
-		return process.stdout.getWindowSize()[1];
-	} : function () {
-		return tty.getWindowSize ? tty.getWindowSize()[0] : 0;
-	}),
-	reset: d.gs(function () {
-		return repeat.call('\n', getHeight() - 1) + '\x1bc';
-	}),
+	trim: d(trim),
+	throbber: d(throbber),
+	width: d.gs(function () { return process.stdout.columns || 0; }),
+	height: d.gs(function () { return process.stdout.rows || 0; }),
+	reset: d.gs(function () { return '\n\x1bc'; }),
```

Comment:
There are some cases where `clc.width()` will returns `0`. Is the case where the terminal that you are using not return this information to process. For instance, Cygwin. On the other hand, it is not supported by Node.

I don't know how I can check if we are running inside Cygwin.